lib/player.rb

```diff
@@ -0,0 +1,61 @@
+require 'awesome_print'
+require 'pry'
+require_relative 'scoring'
+
+module Scrabble
+  class Player < Scoring
+
+    attr_reader :name, :played_words, :player_tiles
+    attr_writer :total_score
+
+    def initialize(name)
+      @player_tiles = []
+      @name = name
+      @played_words = []
+      @total_score = 0
+    end
+
+    def draw_tiles(tile_bag)
+      tile_bag = Scrabble::TileBag.new
```

Comment:
You're getting a `TileBag` instance passed as a parameter you are **NOT** supposed to create a new instance here.  **USE** the parameter!